features/step_definitions/hangout_steps.rb

```diff
@@ -117,10 +118,26 @@
   event = Event.find_by_name(event_name)
   visit event_path(event)
   expect(page).to have_link('Join now', href: @hangout_url)
-  time = Time.parse(@jump_date) + event.duration.minutes - 1.minute
+  time = Time.parse(@jump_date) + event.duration.minutes - 10.minutes
   Delorean.time_travel_to(time)
   visit event_path(event)
   expect(page).to have_link('Join now', href: @hangout_url)
+  time = Time.parse(@jump_date) + event.duration.minutes + 10.minutes
+  Delorean.time_travel_to(time)
+  visit event_path(event)
+  expect(page).not_to have_link('Join now')
+end
+
+When(/^I start the "([^"]*)" event$/) do |event_name|
+  event = Event.find_by_name(event_name)
+  visit event_path(event)
+  find('#liveHOA-placeholder').click
```

Comment:
so this corresponds to clicking on the start hangout button? I don't think that ever caused the event to go live per se.

The event would go live when the hangout plugin in the hangout sent data back to the WSO server and that would make the event go live